42_grml

```diff
@@ -74,10 +74,18 @@ get_dependencies() {
     break
   done
   echo $dependencies
-
 }
 
-
+get_iso_identifier()
+{
```

Comment:
nitpick: please let's keep style consistent and use:

```
get_iso_identified() {
```

(instead of having the "{" in the next line)

Reply:
Whoops, didn't catch that one.